draft-kuehlewind-quic-substrate.md

```diff
@@ -181,32 +181,31 @@ selects the proxy and specifies the requested support function. Often the server
 may not need to be aware of it, however, depending on the optimization function,
 server cooperation could be beneficial as well. However, the client and the proxy
 need a direct and secured communication channel in order to request and configure
-a service and exchange or expose the needed information and metadata. 
+a service and exchange or expose the needed information and metadata.
 
 
-## Frontend Support for Load Balancing and Migration/Mobility 
+## Frontend Support for Load Balancing and Migration/Mobility
 
-In this usage scenario the application service provider aims for flexibility in
-server selection, therefore the client communicates with a reverse proxy that may
-or may not be under the authority of the service provider. Such a proxy assists the client
-to access and select the content requested. Today reverse proxies terminate the
-connection, including the security association, and as such appear as the communication
-endpoint to the client. Terminating not only the transport connection but also the
-security association is especially problematic if the proxy provider is not under the direct
-authority of the actual service provider but a contracted third party.
+The use of proxies can provide benefits to performance, resilience and
+flexibility of access to services.
 
-A similar setup may be used to perform load balancing or migration for mobility support, 
-of both the server or client, where a frontend proxy can redirect the traffic
-to a different backend server. Today this is realized fully transparent to the client 
-and the client is not aware of the network setup behind the proxy. However, such a setup
-may as well benefit in future from an explicit tunneling or proxying approach.
+In one usage scenario the client communicates with a reverse proxy that assists
+with access to and selection of the content requested. This proxy that may or
+may not be under the authority of the service provider. Today such reverse
+proxies terminate the connection, including the security association, and as
+such appear as the communication endpoint to the client. Terminating both
+transport and security may be problematic if the proxy provider is not under the
+direct authority of the actual service provider (e.g. a contracted third party).
 
```

Comment:
Thanks! I think this is good. However, it would be nice to still also mention mobility support somewhere but maybe that's an own paragraph.

Reply:
Mobility is mentioned in the next paragraph, is that not ok?

Reply:
okay